### ISSUE TITLE
PolyPen: Unhandled Exception Caught!

### ISSUE THREAD

**Issue by JimDriscoll** [CGCookie/retopoflow#629]
I'm not used to the PolyPen and was trying to get it to fill in an area with a new face and it gave this error.

-------------------------------------

```
Environment:

- RetopoFlow: 2.0.0
- Blender: 2.79.0 master 2018-03-22
- Platform: Windows, 10, 10.0.17134, AMD64, Intel64 Family 6 Model 79 Stepping 1, GenuineIntel
- GPU: NVIDIA Corporation, GeForce GTX 1070/PCIe/SSE2, 4.6.0 NVIDIA 390.65, 4.60 NVIDIA
- Timestamp: 2018-09-15 14:20:45.167964


Error Hash: 9a42177a85e4c84a725eab0bc42f53aa


Trace:

- EXCEPTION (<class 'ValueError'>): edges.new(): this edge exists
- .../rfmode\rfmesh.py
- 000 1331:new_edge() bme = self.bme.edges.new(verts)
- .../rfmode\rfcontext_target.py
- 001 0396:new_edge() return self.rftarget.new_edge(verts)
- .../rfmode\rftool_polypen.py
- 002 0282:modal_insert() bme = self.rfcontext.new_edge((bmv0, bmv1))
- .../rfmode\rftool.py
- 003 0076:wrapper() ret = fn(*args, **kwargs)
- 004 0110:modal() nmode = self.FSM[self.mode]()
- 005 0114:modal() raise e     # passing on the exception to RFContext
- .../rfmode\rfcontext.py
- 006 0556:modal_main() self.tool.modal()
- 007 0469:modal() nmode = self.FSM[self.mode]()
```

**Comment by vxlcoder** [comment]
Thanks for the report, @JimDriscoll.  Can you by chance post a screen shot or a .blend file?  Or, can you reproduce this error again?

**Comment by JimDriscoll** [comment]
Hi Jon,

Sorry I am unable to reproduce the error. I'm not exactly sure what I was doing. I am trying to get familiar with the RF tools and there was a lot of right clicking with shift and control and some other random key strokes that I might have imagined. I kind of blacked out there for a while... ;-)

All kidding aside. I really love the tool. It just took me a while to get the swing of, but now that I have a bit of experience the tool and it looks like a real time saver. Thank you for creating it.

I've attached the blend file.

Jim

________________________________

**Comment by JimDriscoll** [comment]
Hi Jon,

Sorry I am unable to reproduce the error. I'm not exactly sure what I was doing. I am trying to get familiar with the RF tools and there was a lot of right clicking with shift and control and some other random key strokes that I might have imagined. I kind of blacked out there for a while... ;-)

All kidding aside. I really love the tool. It just took me a while to get the swing of, but now that I have a bit of experience the tool and it looks like a real time saver. Thank you for creating it.

I've attached the blend file.

Sorry if you got this email twice. I reached the attachment limit and wasn't sure if the last resend got to you.

Thanks,
Jim

________________________________

**Comment by vxlcoder** [comment]
Sadly, neither of the attachments saved.

However, this is the same issue as #603, and it should now be fixed in the latest version of `release/2.0.1` branch! If you know how, feel free to test that version, otherwise it will be officially released soon.

I'm closing this issue, but reopen if you continue to have problems.

Thanks for reporting this issue and being willing to post your .blend file!
